x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilties.test.tsx

```diff
@@ -131,12 +131,12 @@ describe('<Vulnerabilities />', () => {
     (useCspIntegrationLink as jest.Mock).mockImplementation(() => chance.url());
 
     renderVulnerabilitiesPage();
-
+    // TODO: Add back Index timeout check later
```

Comment:
Why not now? Not sure I fully understand the comment

Reply:
I see
So I suggest that we will manage that as part of the DOD of the future task instead of using TODO comments in the code. 
That way, when delegating the task we won't miss adding a test case